code/modules/mob/living/carbon/alien/say.dm

```diff
@@ -15,11 +15,21 @@
 	if(copytext(message,1,2) == "*")
 		return emote(copytext(message,2))
 
+	var/needtohack = 1 //hacky way to get around stupid byond shit
 	var/datum/language/speaking = parse_language(message)
+
+	if(speaking == null)
+		var/quickcheck = get_default_language()
+		if(quickcheck == null)
+			speaking = all_languages["Xenomorph"] //this is the hackiest thing, no clue how to avoid it
+			needtohack = 0
+		else
+			speaking = get_default_language()
+			needtohack = 0
+
 	if(speaking)

```

Comment:
@GinjaNinja32 But I do have another problem that I just noticed while testing this- the default languages work fine, buuut silicons check-known-languages menu doesn't actually work. As in, it can speak basically any language, but they don't show up in it's menu.